content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -120,3 +180,5 @@ The following configuration options behave consistently across the latest versio
 [1]: /developers/community/libraries/#apm-tracing-client-libraries
 [2]: /tracing/trace_collection/compatibility/java/#framework-integrations-disabled-by-default
 [3]: /tracing/services/inferred_services/
+[4]: /tracing/trace_pipeline/ingestion_mechanisms/
+[5]: /tracing/runtime_metrics/
```

Comment:
The previous `/tracing/runtime_metrics/` didn't work for me, but I think this should:

```suggestion
[5]: /tracing/metrics/runtime_metrics/
```